### PR TITLE
RHINENG-10058: Another attempt at fixing invalid plan names...

### PR DESCRIPTION
**Pull request by rexwhite**
Trying this again, the old-school way...

### CODE REVIEW

File: db/migrations/20240517140534-avoid-tabs-with-postgres.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+const _ = require('lodash');
+
+// Fix remediation plan names that have leading/trailing tabs
+//
+// Something about the combination of Sequelize and postgres makes sql containing \t's behave unpredictably,
+// so we're doing this migration again, the tedious way...
+exports.up = async (q) => {
+    // get list of remediation plans
+    const [plans, metadata] = await q.sequelize.query("SELECT name, tenant_org_id, id FROM remediations;");
```
Comment (vkrizan) [inline db/migrations/20240517140534-avoid-tabs-with-postgres.js:11]: Could this return many records that could lead to a memory limit hit?

Comment (rexwhite) [reply db/migrations/20240517140534-avoid-tabs-with-postgres.js:11]: Yeah, this was sort of "work in progress" and is deficient in several respects - updated code coming shortly!

File: db/migrations/20240517140534-avoid-tabs-with-postgres.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        plan.name = plan.name.trim();
+
+        // check for collisions with existing plans
+        let dup_count = 0;
+        let new_name = plan.name;
+
+        while (await collides(new_name, plan.tenant_org_id)) {
+            dup_count++;
+            new_name = plan.name + ` - ${dup_count}`;
+        }
```
Comment (marleystipich2) [inline db/migrations/20240517140534-avoid-tabs-with-postgres.js:34]: This is a good idea!

**Review by marleystipich2 — APPROVED** [review]
LGTM!

**Pull request merged**